File: pset6/mario/mario.py
import sys
import cs50


def main():
    while True:
        print("Pyramid Height: ", end="")
        # n = sys.stdin.readline()
        # n = int(n)
        n = cs50.get_int()
        if (n >= 0) and (n <= 23):
            break

    for i in range(0, n):
        for j in range(0, n - (i + 1)):
            print(" ", end="")
        for m in range(0, i + 1):
            print("#", end="")
        print("  ", end="")
        for k in range(0, i + 1):
            print("#", end="")
        print()


if __name__ == "__main__":
    main()

File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(){
    int n;

    do
    {
        printf("Please specify the height of the pyramid: ");
        n=get_int();
    } while(n<0||n>23);

    for(int i=0;i<n;i++){
        for(int j=0;j<n-(i+1);j++){
            printf(" ");
        }
        for(int j=0;j<i+1;j++){
            printf("#");
        }
        printf("  ");
        for(int k=0;k<i+1;k++){
            printf("#");
        }

        printf("\n");
    }
}